recipes/coin/meta.yaml

```diff
@@ -0,0 +1,57 @@
+package:
+  name: coin
+  version: 4.0.0
+
+source:
+  fn: coin.tar.gz
+  url: https://bitbucket.org/Coin3D/coin/get/CMake.tar.gz
```

Comment:
I don't think there is. This cmake-branch is quite new, but simplify the build process a lot. Maybe it's possible to use the latest commit as url?